scripts/qa_crowbarsetup.sh

```diff
@@ -1754,6 +1770,15 @@ function onadmin_bootstrapcrowbar()
             cat setup-database.txt
             complain 36 "Could not setup PostgreSQL database"
         fi
+
+        if [[ $migrate_db = 1 ]] ; then
+            http_code=`curl -s -X POST -H "Accept: application/vnd.crowbar.v2.0+json" -o migrate-database.txt -w '%{http_code}' $crowbar_init_api/database/migrate`
+            if ! [[ $http_code =~ [23].. ]] ; then
+                cat migrate-database.txt
+                complain 36 "Could not migrate the Crowbar database"
+            fi
+        fi
+
         # FIXME: when https://github.com/SUSE-Cloud/automation/pull/1105 is merged
         http_code=`curl -s -X POST -H "Accept: application/vnd.crowbar.v2.0+json" -o initialize-crowbar.txt -w '%{http_code}' $crowbar_init_api/init`
```

Comment:
Not sure, but I think we don't need to run install-chef-suse after the upgrade. It's already installed...